src/main/scripts/upgrade_oracle_5_0.sql

```diff
@@ -55,6 +55,23 @@ ALTER TABLE DATASET ADD FILESIZE NUMBER(19);
 ALTER TABLE INVESTIGATION ADD FILECOUNT NUMBER(19);
 ALTER TABLE INVESTIGATION ADD FILESIZE NUMBER(19);
 
+-- Add DataPublicationType table
+
+CREATE TABLE DATAPUBLICATIONTYPE (
+	ID NUMBER(19) NOT NULL, 
+	CREATE_ID VARCHAR2(255) NOT NULL,
+	CREATE_TIME TIMESTAMP NOT NULL,
+	DESCRIPTION VARCHAR2(255), 
+	MOD_ID VARCHAR2(255) NOT NULL,
+	MOD_TIME TIMESTAMP NOT NULL,
+	NAME VARCHAR2(255) NOT NULL, 
+	FACILITY_ID NUMBER(19) NOT NULL, 
+	PRIMARY KEY (ID)
+);
+
+ALTER TABLE DATAPUBLICATIONTYPE ADD CONSTRAINT FK_DATAPUBTYPE_FACILITY_ID FOREIGN KEY (FACILITY_ID) REFERENCES FACILITY (ID);
+ALTER TABLE DATAPUBLICATIONTYPE ADD CONSTRAINT UNQ_DATAPUBLICATIONTYPE_0 UNIQUE (FACILITY_ID, NAME);
```

Comment:
Same effect as for `DATACOLLECTIONINVESTIGATION` above, I get the following definition of `DATAPUBLICATIONTYPE` after upgrade:
```sql
CREATE TABLE `DATAPUBLICATIONTYPE` (
  `ID` bigint(20) NOT NULL AUTO_INCREMENT,
  `CREATE_ID` varchar(255) NOT NULL,
  `CREATE_TIME` datetime NOT NULL,
  `DESCRIPTION` varchar(255) DEFAULT NULL,
  `MOD_ID` varchar(255) NOT NULL,
  `MOD_TIME` datetime NOT NULL,
  `NAME` varchar(255) NOT NULL,
  `FACILITY_ID` bigint(20) NOT NULL,
  PRIMARY KEY (`ID`),
  UNIQUE KEY `UNQ_DATAPUBLICATIONTYPE_0` (`FACILITY_ID`,`NAME`),
  CONSTRAINT `DATAPUBLICATIONTYPEFACILITY_ID` FOREIGN KEY (`FACILITY_ID`) REFERENCES `FACILITY` (`ID`),
  CONSTRAINT `FK_DATAPUBLICATIONTYPE_FACILITY_ID` FOREIGN KEY (`FACILITY_ID`) REFERENCES `FACILITY` (`ID`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb4;
```